bots/tests-invoke

```diff
@@ -52,10 +52,14 @@ def main():
     name = os.environ.get("TEST_NAME", "tests")
     revision = os.environ.get("TEST_REVISION")
     test_project = os.environ.get("TEST_PROJECT")
+    # Set up to branch when overwritten with test name
+    # e.g. PR opened against master for test `rhel-8-0@cockpit-project/cockpit/rhel-8.0`
+    # TEST_BRANCH would be `rhel-8.0`
+    test_branch = os.environ.get("TEST_BRANCH", "")
```

Comment:
Could this keep the default `None` when not given? This makes errors in the code easier to spot, as we really don't want to export the variable with an empty value.

Reply:
Sure, fixed